webapp/src/shared/components/Row/Row.tsx

```diff
@@ -15,38 +17,46 @@ export type RowProps = {
 export default function Row({
   iconVariant,
   onClick,
+  url,
   avatarProps,
   title,
   subtitle,
 }: RowProps) {
-  return (
-    <button className={`row paragraph-regular`} onClick={onClick}>
-      {avatarProps && <SmallAvatar {...avatarProps} />}
-      <div className={`row_text_wrapper`}>
-        {title && (
-          <Text
-            variant={TextVariant.SUBHEADING}
-            color={TextColor.LIGHT}
-            weight={TextWeight.BOLD}
-          >
-            {title}
-          </Text>
-        )}
-        {subtitle && (
-          <Text
-            variant={TextVariant.PARAGRAPH}
-            color={TextColor.LIGHT}
-            weight={TextWeight.REGULAR}
-          >
-            {subtitle}
-          </Text>
-        )}
-      </div>
-      {iconVariant && (
-        <div className="row-icon">
-          {<Icon variant={iconVariant} size={IconSize.SMALL} />}
-        </div>
+  const RowChildren: JSX.Element[] = [
+    (avatarProps && <SmallAvatar {...avatarProps} />) || <div />,
```

Comment:
suggestion: Here we could use a [fragment](https://reactjs.org/docs/fragments.html#short-syntax). This way, we could omit the empty divs.